### PR TITLE
Honeywell N95s Added to Store

### PR DESCRIPTION
**Pull request by mekkim**
Added Honeywell NIOSH N95-small masks to store

### CODE REVIEW

File: src/views/Store.js [hunk 1]
```diff
@@ -258,6 +260,54 @@ const BuyPage = () => {
             </Card>
           </Col>
         </Row>
+        <hr />
+		
+		<Row>
```
Comment (humphd) [inline src/views/Store.js:265]: Nit: fix indenting here

File: src/views/Store.js [hunk 2]
```diff
@@ -258,6 +260,54 @@ const BuyPage = () => {
...
+          <Col md="8">
+            <Card className="borderless">
+              <CardBody>
+                <CardImg
+                  width="720"
```
Comment (humphd) [inline src/views/Store.js:270]: Change this to `495` to match the actual image width

**Review by humphd — CHANGES_REQUESTED** [review]
Looks great.  Two small things.

**Review by humphd — APPROVED** [review]
👍

**Comment by mekkim** [comment]
Resolves #137

**Comment by mekkim** [comment]
Resolves #137

**Pull request merged**